src/frontend/pages/products/Products.jsx

```diff
@@ -1,12 +1,21 @@
 import React, { useState, useEffect } from "react";
 import { Navbar, ProductCard } from "../../components";
 import { Filters } from "./Filters";
+import { useFilter } from "../../contexts";
+import {
+	ComposeFunction,
+	sortbyPriceFunction,
+	filterByPrice,
+	filterByRatings,
+	filterByCategory,
+} from "../../utils/filters";
 import "./products.css";
 
 const axios = require("axios").default;
```

Comment:
is there any specific reason for not directly importing Axios?

Reply:
no will change it later